validate_1.0.0-rc.2/README.md

```diff
@@ -66,7 +67,24 @@ The expected output from running the sample is as follows:
 		-------------------------------------------------------------------------------------
 
 		VALID OCSF.
-		
+
+## Checksum <a name="Checksum"></a>
+
+To ensure the soundness of the download, you may choose run a checksum against the files in this repository.
+
+In order to generate the expected hash and run the checksum please run the following:
+
+	tar -czvf hash.tar.gz /path/to/validate_1.0.0-rc.2/ setup.py validate.py sample requirements.txt sample README.md ocsf_schema_1.0.0-rc.2 docs LICENSE
```

Comment:
Is this tested? Also, adding in /path/to/validate_1.0.0-rc.2/ seems unnecessary and won't allow for direct copy-paste, instead why not have cd statement in the first line and make everything else copy-paste-able?

Reply:
this has been tested, copy and paste commands have been implemented